### DISCUSSION TITLE
Source Not Available in Plots

### DISCUSSION THREAD

**Discussion by afaehn** [danielhrisca/asammdf#630] [Q&A]
Hey Everyone, I'm a bit new to ASAMMDF, so I'm hoping this is a pretty easy answer. If you look at the image attached, I'm looking to populate this field in the plot area, but I am unable to figure out how to populate this. My assumption is that this as to come from the DBC file itself. I have added several attributes to the DBC file without luck in getting this to populate. Any ideas?
![GPSMod Source Not Available](https://user-images.githubusercontent.com/97107030/148070404-c4949efa-3772-4ef2-8514-b1c86c9f9b4d.JPG)

**Comment by danielhrisca** [comment]
This looks cool.

What exactly did you expect to see there in the source field?

**Reply by afaehn** [reply]
Good question. I was hoping to see the source device that this came from. Currently, I have several DBC files for different devices i use during testing. I also have DBC files for each of my customers CAN bus networks. If possible, I would like to enter the name of the device that this signal came from into the DBC, and have ASAMMDF read it. So in this case, maybe it could read "CSSElectronics_GPSMod"?
